reltoken.php

```diff
@@ -247,6 +247,24 @@ function reltoken_civicrm_install() {
  * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_postInstall
  */
 function reltoken_civicrm_postInstall() {
+  // Get the custom field ID of the field that specifies generating tokens.
+  $tokenCustomFieldId = civicrm_api3('CustomField', 'getvalue', [
+    'name' => 'display_reltokens',
+    'return' => 'id',
+  ]);
+
+  // add generating tokens in RelationshipType
+  $result = civicrm_api3('RelationshipType', 'get', [
+    'sequential' => 1,
+    'is_active' => 1,
+    'api.RelationshipType.create' => [
+      'name_a_b' => '$value.name_a_b',
+      'name_b_a' => '$value.name_b_a',
```

Comment:
@lucky091588 Seems redundant to set `name_a_b` and `name_b_a`. Is that necessary?

Reply:
@twomice It is necessary since it is a mandatory field when using RelationshipType create in api3.

Reply:
@twomice I just confirmed that it's not necessary since it has the `id` with a value which means updating.